medicinal_plants/lib/main.dart

```diff
@@ -1,6 +1,7 @@
 // ignore_for_file: prefer_const_constructors
 
 import 'package:flutter/material.dart';
+import 'package:medicinal_plants/src/screens/home-screen.dart';
```

Comment:
Use relative imports instead of absolute import:

something like: 'src/screens/home-screen.dart' or 'screens/home-screen.dart'
whichever is applicable